plugins/doc_fragments/check_mode.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
Are we adding this as a separate doc fragment because some modules don't need check_mode?

Reply:
I think they all need check_mode but currently they don't all support it and I didn't want to add that support in this PR.